source/expressions.tex

```diff
@@ -3995,7 +3995,7 @@
 No temporary is created, no copy is made, and
 no constructors\iref{class.ctor} or conversion
 functions\iref{class.conv} are called.%
-\footnote{This is sometimes referred to as a \defn{type pun}
+\footnote{This is sometimes referred to as a type pun
```

Comment:
We're losing "type pun" from the index. That's an undesirable regression.

Reply:
Should we just add an index entry above the paragraph?

Reply:
Yes, that would address my concerns.